### PR TITLE
doc: add contribution guides

### PR DESCRIPTION
**Pull request by esatterwhite**
Includes a standard contributing and code of conduct document to
eastablish a common ground of expectations from the community and from
ourselves

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+## Process
+
+We use a fork-and-PR process, also known as a triangular workflow. This process
+is fairly common in open-source projects. Here's the basic workflow:
+
+1. *Fork* the upstream repo to create your own repo. This repo is called the origin repo.
+2. *Clone* the origin repo to create a working directory on your local machine.
```
Comment (smusali) [inline CONTRIBUTING.md:9]: @esatterwhite, why are these verbs in `1.` and `2.` in `italic` but not others?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+If this workflow is too much to understand to start, that's ok! You can use
+GitHub's UI to make a change, which is autoset to do most of this process for
+you. We just want you to be aware of how the entire process works before
+proposing a change.
+
+Thank you for your contributions; we appreciate you!
+
+## License
+
+Note that we use a standard MIT license on this repo.
```
Comment (smusali) [inline CONTRIBUTING.md:30]: @esatterwhite, should we use the link to [the license file](https://github.com/logdna/python/blob/master/LICENSE)?

Comment (smusali) [reply CONTRIBUTING.md:30]: @esatterwhite, by the way, just noticed; the license is outdated - it should be updated

File: CONTRIBUTING.md (hunk 1 above)
Comment (tanberry) [reply CONTRIBUTING.md:9]: Errrr that was me being picky. Trying to, well, emphasize the two words. ;-) Feel free to remove the asterisks and free the verbs if you like.

**Review by tanberry — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (esatterwhite) [reply CONTRIBUTING.md]: removed

File: CONTRIBUTING.md (hunk 2 above)
Comment (esatterwhite) [reply CONTRIBUTING.md]: Linked + updated. Set copyright year to 2017 as it should be set to the year of original publication. That was the year of the first release on PyPi

**Review by smusali — APPROVED** [review]
lgtm!

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 LogDNA
+Copyright (c) 2017 LogDNA
```
Comment (david-ly) [inline LICENSE:3]: Why are we using 2017 here?

**Review by david-ly — COMMENTED** [review]
lgtm, just had a question about the license

File: LICENSE (hunk 3 above)
Comment (david-ly) [reply LICENSE:3]: Nvm, resolving this was answered in a previous conversation `Set copyright year to 2017 as it should be set to the year of original publication. That was the year of the first release on PyPi`

**Review by david-ly — APPROVED** [review]

**Review by mdeltito — APPROVED** [review]

**Pull request merged**